docs/docusaurus.config.js

```diff
@@ -0,0 +1,117 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'UrTurn',
+  tagline: 'Quickly build multiplayer games - only four functions!',
+  url: 'https://docs.urturn.app',
```

Comment:
base url changed from `turnbasedgames.github.io`, reasoning for this is in the pr description.